docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -72,6 +72,33 @@ Dependency search provides the following filters, which correspond to the data p
 ![Screenshot of dependency search with query](/img/SSC-DepSearch-Query.png#md-width)
 _**Figure**. Dependency search page with sample search query._
 
+## Dependency graphs (beta)
+
+:::info
+Semgrep does not support the generation of dependency graphs for managed scans of projects.
+:::
+
+Dependency graphs allow you to view dependency paths for all transitive dependencies, up to seven layers deep, introduced in a project. With this information, you can understand how a transitive dependency was introduced and how deep it is nested in the dependency tree reflecting your project that Semgrep generates.
```

Comment:
```suggestion
Dependency graphs allow you to view dependency paths for all transitive dependencies, up to seven layers deep, introduced in a project. With this information, you can understand how a transitive dependency was introduced and how deep it is nested in the dependency tree reflecting your project that Semgrep generates.
```
The clause, "...how deep it is nested in the dependency tree reflecting your project that Semgrep generates," is hard to understand - maybe break down into several sentences?